file.c

```diff
@@ -93,12 +93,23 @@ void loadImage(const char *filename, AVFrame **image) {
     if (pkt.stream_index != 0)
         errOutput("unable to open file %s: invalid stream.", filename);
 
+    while (!got_frame && pkt.data) {
+
+        if (pkt.size <= 0) {
+            pkt.data = NULL;
+            pkt.size = 0;
+        }
+
     ret = avcodec_decode_video2(avctx, frame, &got_frame, &pkt);
```

Comment:
Hi Flameeyes! Please be so kind and make a new patch on top of master. My PR is against the 6.1 and not against master. And I've no practical experience with C and it's a headache for me to get a dev environment with all necessary tools set up.
Thank you!